sdk/eventhub/event-hubs/src/eventData.ts

```diff
@@ -194,7 +194,7 @@ export namespace EventDataInternal {
    * Converts an EventData object to an AMQP message.
    * @param {EventDataInternal} data The EventData object that needs to be converted to an AMQP message.
    */
-  export function toAmqpMessage(data: EventDataInternal): Message {
+  export function toAmqpMessage(data: EventDataInternal, batchLabel?: string): Message {
```

Comment:
At this point, we are only aware of the given message and the need to convert it to an amqpMessage. The concept of "batch" is external to this method. Let's use the name `partitionKey` here as that is the word we use when setting the message_annotations anyway

Also, please update the jsdocs for the second parameter